models/Hat.cs

```diff
@@ -15,5 +15,12 @@ public class Hat
         public int dayFrom { get; set; }
         public int monthTo { get; set; }
         public int dayTo { get; set; }
+
+        /// <summary>
+        ///     Weight that determines the chance of this hat getting chosen.
+        ///     Increase the number to have this hat chosen more common, decrease it to make it more rare.
+        /// </summary>
+        /// <value>A positive number representing the rarity of the hat. Defaults to 1.</value>
+        public double weight { get; set; } = 1;
```

Comment:
In the same vein as this - can you make Enabled true by default - remove Enabled from those where it's already true